include/openssl/ui.h

```diff
@@ -266,14 +266,12 @@ DEFINE_STACK_OF(UI_STRING)
  * The different types of strings that are currently supported. This is only
  * needed by method authors.
  */
-enum UI_string_types {
-    UIT_NONE = 0,
-    UIT_PROMPT,                 /* Prompt for a string */
-    UIT_VERIFY,                 /* Prompt for a string and verify */
-    UIT_BOOLEAN,                /* Prompt for a yes/no response */
-    UIT_INFO,                   /* Send info to the user */
-    UIT_ERROR                   /* Send an error message to the user */
-};
```

Comment:
It's on my list for a future api-breaking release.  We made things opaque knowing that we'd miss things and we'd fix it as bugs.  This is a little- to not-used API that I want to ABI-compatibly make safer.